platform/mellanox/sdk-src/sx-acl-helper/Makefile

```diff
@@ -0,0 +1,29 @@
+.ONESHELL:
+SHELL = /bin/bash
+
+MAIN_TARGET = sx-acl-helper_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
+DERIVED_TARGETS = sx-acl-helper-dev_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb \
+				  sx-acl-helper-dev-static_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb \
+				  sx-acl-helper-dbgsym_1.mlnx.$(MLNX_SDK_DEB_VERSION)_amd64.deb
+PACKAGE_NAME = sx_acl_helper
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# get sources
+	rm -rf $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
+
+	wget -c $(MLNX_SDK_SOURCE_BASE_URL)/$(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION).tar.gz -O - | tar -xz
+
+	# build
+	pushd $(PACKAGE_NAME)-$(MLNX_SDK_VERSION)-$(MLNX_SDK_ISSU_VERSION)
+
+	if [ -f autogen.sh ]; then
+		./autogen.sh
+	fi
+
+	debuild -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS)
+
+	popd
+
+	mv $(DERIVED_TARGETS) $* $(DEST)/
```

Comment:
>$* [](start = 23, length = 2)

Could you explain `$*`?

Reply:
Hi, it stands for the default deb package. The sx-acl-helper_1.mlnx.4.3.2104_amd64.deb, in this case.